### DISCUSSION TITLE
Rule ingest/upgrade to sigma format

### DISCUSSION THREAD

**Discussion by EbolaWare** [SigmaHQ/pySigma#98] [Q&A] [answered]
I'm currently working to pull in existing detection rules in other systems and convert them to sigma. The systems are acceptable outputs for sigma.

**Are there any tools to assist in this process?** I'd really like to stop hating myself right about now. I'm ready to write my own solution, but I know this is a one off for me.

**Comment by thomaspatzke** [comment, marked as answer]
The old version of [uncoder.io](https://uncoder.io/) was able to do this, but doesn't seems to be available anymore. There's no SigmaHQ open source project for doing this.
